src/ImageSharp/Formats/Jpeg/JpegDecoderCore.cs

```diff
@@ -506,90 +506,48 @@ public void Dispose()
         /// Returns the correct colorspace based on the image component count and the jpeg frame component id's.
```

Comment:
The comment is no longer valid, maybe change it like this:
```suggestion
        /// Returns the correct colorspace based on the image component count and adobe APP14 marker's color transform flag.
```